.sync.yml

```diff
@@ -2,4 +2,4 @@
 Rakefile:
   param_docs_pattern:
     - manifests/init.pp
-    - manifests/plugin/*.pp
+    - manifests/plugin/**/*.pp
```

Comment:
I don't think this is correct. Now it will also require docs for all ```params.pp``` which is undesired.